config/routes.rb

```diff
@@ -4,7 +4,7 @@
 APIs to query approval service
 
 OpenAPI spec version: 1.0.0
-
+Contact: support@redhat.com
```

Comment:
Do we want redhat.com email addresses in any of the open source versions of this repo?

Reply:
I am not sure, just referred to service portal's swagger yaml.